registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/response/dto/Analytics.java

```diff
@@ -0,0 +1,11 @@
+package io.mosip.registration.processor.manual.verification.response.dto;
+
+import lombok.Data;
+
+@Data
+public class Analytics {
+	
+	private String key1 = "value1";
```

Comment:
there could be multiple key values which could be more than 2. can we make it dynamic